src/browser/browser.zig

```
@@ -132,9 +136,25 @@ pub const Session = struct {
         self.alloc.destroy(self);
     }
 
+    pub fn setInspector(
+        self: *Session,
+        ctx: *anyopaque,
+        onResp: jsruntime.InspectorOnResponseFn,
+        onEvent: jsruntime.InspectorOnEventFn,
+    ) !void {
+        self.inspector = try jsruntime.Inspector.init(self.alloc, self.env, ctx, onResp, onEvent);
+        self.env.setInspector(self.inspector.?);
+    }
+
     pub fn createPage(self: *Session) !Page {
         return Page.init(self.alloc, self);
     }
+
+    pub fn callInspector(self: *Session, msg: []const u8) void {
+        if (self.inspector) |inspector| {
```

I would panic if the inspector is missing.